client/src/components/Toolbar/Menu.js

```diff
@@ -0,0 +1,34 @@
+import React, {Fragment} from "react";
+import { SideNav, SideNavItem,} from  "react-materialize";
+import 'materialize-css/dist/css/materialize.min.css';
+
+
+const Menu = () => {
+ return (
+   <Fragment>
+     <SideNav
+       trigger={
+         <button>
+           <img src={require("../../images/menu.png")} alt="hamburgerIcon"/>
+         </button>
+       }
+       options={{ closeOnClick: true }}
+     >
+     <img src={require("../../images/user.png")} alt="user"/>
+       <SideNavItem
+         userView
+         user={{
+           background: "",
+
+           name: "John Doe",
```

Comment:
I imagine these are temporary inputs, but you might find it easier to deal with these sort of entries by putting them as constants at the top of the file, such as: `const TEMPORARY_NAME = "John Doe";`.  That way they are easily identifiable for future maintenance and also easy to search for.